ecs_fargate/README.md

```diff
@@ -102,15 +125,17 @@ The ECS Fargate check does not include any events at this time.
 
 ### Service Checks
 
-The ECS Fargate check does not include any service checks at this time.
+**fargate_check**  
+Returns `CRITICAL` if the Agent is unable to connect to Fargate, otherwise returns `OK`.
```

Comment:
👌

Reply:
@hkaj - According to the service setup screen, Fargate uses REPLICA service type - https://cl.ly/eabb4026dd19